doc/conf.py

```diff
@@ -112,7 +128,8 @@
     'gallery_dirs': ['auto_examples', 'auto_tutorials'],
     'within_subsection_order': FileNameSortKey,
     # Settings for linking between examples & API examples
-    'backreferences_dir': 'backrefs',
+    'backreferences_dir': 'generated',
     'doc_module': ('neurodsp',),
-    'reference_url': {'neurodsp': None}
+    'reference_url': {'neurodsp': None},
+    'remove_config_comments': True
```

Comment:
this looks like a cool config option that I didn't know! Good find!

My understanding is that this should remove things like `sphinx_gallery_thumbnail_number = 2` from the bottom of the sphinx gallery files. However, when I build locally, it seems this isn't working? Not sure what's going on with it - but can you check if you can get it working?

Reply:
These lines have been moved to the top import code blocks.